qdrant_client/local/distances.py

```diff
@@ -147,13 +147,21 @@ def calculate_distance_core(
         return calculate_distance(query, vectors, distance_type)
 
 
+def fast_sigmoid(x: np.float32) -> np.float32:
+    if not np.isfinite(x):
```

Comment:
I am a bit confused by it, I know, that we have had such a check for quite some time now though..

Do we wait for a particular not finite type here?

Like -inf, +inf, nan ? (if we wait -inf, won't it be better to return a constant value?)

It seems for me that at the moment, If vector has a non-finite difference with any of context pairs, then it is excluded from result search, because it will have non-finite score in `overall_scores`

Reply:
Yeah, I don't know if I understand well, but if x equal +inf or -inf the function return x is good or you want to return for example 2 or 3 ?